ops/terraform/services/migrator/deploy.groovy

```diff
@@ -70,6 +70,11 @@ boolean deployMigrator(Map args = [:]) {
             parameterType: "String",
             shouldOverwrite: true
         )
+        awsSsm.tagResource(
```

Comment:
where is this defined? I'm guessing you meant to create it instead of having its functionality inside of `awsSsm.putParameter`?

Reply:
good find. I need to remove this. Thanks!

Reply:
I do think this should be its own global function, but I would call it inside `awsSsm.putParameter` instead of explicitly elsewhere